pkg/k8shandler/clusterloggingrequest_test.go

```diff
@@ -0,0 +1,40 @@
+package k8shandler
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+)
+
+var _ = Describe("ClusterLoggingRequest", func() {
+
+	var (
+		clr *ClusterLoggingRequest
+	)
+	BeforeEach(func() {
+		clr = &ClusterLoggingRequest{
+			Cluster: &logging.ClusterLogging{
+				Spec: logging.ClusterLoggingSpec{},
+			},
+		}
+
+	})
+	Describe("#IncludesManagedStorage", func() {
```

Comment:
Does '#' mean something or is it a typo?

Reply:
It's only a means to chain the describes together so its more obvious we are referring to a given type and method `ClusterLoggingRequest#IncludesManagesStorage`